assets/js/modules/optimize/setup.js

```diff
@@ -295,11 +296,10 @@ class OptimizeSetup extends Component {
 
 				<div className="googlesitekit-setup-module__inputs">
 					<TextField
-						className={ `
-							mdc-text-field
-							${ errorCode ? 'mdc-text-field--error' : '' }
-							${ OptimizeIDValidated ? '' : 'mdc-text-field--error' }
-						` }
+						className={ classnames(
+							'mdc-text-field',
+							{ 'mdc-text-field--error': errorCode || OptimizeIDValidated }
```

Comment:
This conditional is incorrect. The original conditional rendered this if `errorCode` was truthy or `OptimizeIDValidated` was falsey. This should be: `{ 'mdc-text-field--error': errorCode || ! OptimizeIDValidated }`